content/docs/extensions/model/sklearn.md

```diff
@@ -0,0 +1,69 @@
+# Scikit-Learn Models Support
+
+[ModelType](/doc/user-guide/mlem-abcs#modeltype) implementations for any
+sklearn-compatible classes as well as `Pipeline`
+
+## Description
+
+**TODO**
+
+## Requirements
+
+```bash
+pip install mlem[sklearn]
+# or
+pip install scikit-learn
+```
+
+## Examples
+
+### Saving and loading Scikit-Learn model
+
+```python
+from sklearn.datasets import load_iris
+from sklearn.ensemble import RandomForestClassifier
+
+from mlem.api import save, load
+
+
+data, y = load_iris(return_X_y=True, as_frame=True)
```

Comment:
I wish we could auto-generate this from our tests, but not sure it's possible. We have asserts, fixtures, etc, it's going to be a challenge to remove them and make the code both working and readable.